testing/trino-testing-containers/src/main/java/io/trino/testing/containers/junit/ReportLeakedContainers.java

```diff
@@ -81,6 +81,8 @@ public void testPlanExecutionFinished(TestPlan testPlan)
 
             List<Container> containers = dockerClient.listContainersCmd()
                     .withLabelFilter(Map.of(DockerClientFactory.TESTCONTAINERS_SESSION_ID_LABEL, DockerClientFactory.SESSION_ID))
+                    // ignore status "exited" - for example, failed containers after using `withStartupAttempts()`
+                    .withStatusFilter(List.of("created", "restarting", "running", "paused"))
```

Comment:
why isn't "starting" on the list?

Reply:
I based this on the Javadoc for `withStatusFilter`: https://github.com/docker-java/docker-java/blob/main/docker-java-api/src/main/java/com/github/dockerjava/api/command/ListContainersCmd.java#L78
This matches the CLI docs: https://docs.docker.com/reference/cli/docker/container/ls/#filter

Do you have a better reference?

Reply:
The set of status can change in the future, right?

Reply:
Yes, although I'm not sure how likely is that. This list hasn't changed at least since 8 years: https://github.com/docker-java/docker-java/commit/acfe65de6f4ef1392a9bc561a01d88268ef9869b

Reply:
Java API javadoc may or may not be up to date. It's not the source of truth for docker container statuses.

Reply:
Sure, do you have a better reference?

Reply:
No, but also i didn't write this code :)